scripts/apps/workspace/content/components/ContentProfileFieldsConfig.tsx

```diff
@@ -226,7 +231,10 @@ class ItemComponent extends React.PureComponent<IPropsItem> {
 class ItemsContainerBase extends React.PureComponent {
     render() {
         return (
-            <div className="sd-list-item-group sd-list-item-group--space-between-items sd-padding-x--2 sd-padding-y--3">
+            <div
+                className="sd-list-item-group sd-list-item-group--space-between-items sd-padding-x--2 sd-padding-y--3"
+                data-test-id="content-profile-items-list"
```

Comment:
Avoid word `item` because it can mean anything. In this case - content profiles have fields so you could call `content-profile-fields`